---
title: Regulatory genomics
---

Complex organisms consist of hundreds of different celltypes. Under the assumption that every cell contains a more or less identical copy of the same genome, the main difference between tissues consists in variable sets of genes, which are active at a given tissue and developmental stage. In collaboration with our wet lab group, we study how different expression profiles, that can be observed in adult tissues are established during development. Using high-througput technologies like second generation sequencing and microarrays together with genomewide data sources  obtained from various databases, we study regulatory networks of transcription factors, micro RNAs, distant cis-regulatory elements, and their target genes in the context of vertebrate development and responses to exteranl stimuli.

**Protein-Protein Interaction Networks and Their Relationship to the Diseasome**

The word diseasome has been coined to describe the network of genes and proteins whose function is altered in hereditary (and acquired) disease. It is widely assumed that we will need to understand the network properties of cellular (metabolic, transcriptional, protein-protein interaction) networks in order to fully understand disease and to design optimal treatments. This topic is an important focus of our work. We have recently shown that global network characteristics, as measured by random-walk or diffusion kernel analysis, can be harnessed to improve candidate-gene prioritization for positional cloning projects.

We have also applied this approach to the interpretation of exome sequencing data.

