---
title: Selected publications
---


<ol class="bibliography">
 
 <li><span id="PMID27569544"><b>Jäger M</b>, <b>Schubach M</b>, <b>Zemojtel T</b>, Reinert K, Church DM, <b>Robinson PN</b> (2016) Alternate-locus aware variant calling in whole genome sequencing. <i>Genome Medicine </i>, .</span> <a href="https://www.ncbi.nlm.nih.gov/pubmed/27964746">Link</a></li>
<li><span id="PMID27569544"> Smedley D, <b>Schubach M</b>, Jacobsen JO, <b>Köhler S</b>, <b>Zemojtel T</b>, Spielmann M, <b>Jäger M</b>, Hochheiser H, Washington NL, McMurry JA, Haendel MA, Mungall CJ, Lewis SE, Groza T, Valentini G, <b>Robinson PN</b> (2016) Whole-Genome Analysis Framework for Effective Identification of Pathogenic Regulatory Variants in Mendelian Disease. <i>American Journal of Human Genetics</i>, .</span> <a href="https://www.ncbi.nlm.nih.gov/pubmed/27569544">Link</a>  </li>
<li><span id="exomiser_natgenet"> Smedley D, Jacobsen JO, <b>Jäger M</b>, <b>Köhler S</b>, <b>Holtgrewe M</b>, <b>Schubach M</b>, Siragusa E, <b>Zemojtel T</b>, Buske OJ, Washington NL, Bone WP, Haendel MA, <b>Robinson PN</b>. (2015). Next-generation diagnostics and disease-gene discovery with the Exomiser. <i>Nature Protocols</i>, .</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/26562621">Link</a>  </li>
<li><span id="natrevgenet"> Brookes AJ, <b>Robinson PN</b> (2015). Human genotype-phenotype databases: aims, challenges and opportunities. <i>Nature Reviews Genetics</i>, .</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/26553330">Link</a>  </li>  
<li><span id="phenolyzer"> Yang H, <b>Robinson PN</b>, Wang K. (2015). Phenolyzer: phenotype-based prioritization of candidate genes for human diseases. <i>Nature Methods</i>, .</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/26192085">Link</a>  </li>
<li><span id="ajhg">Groza, T., <b>Kohler, S.</b>, Moldenhauer, D., Vasilevsky, N., Baynam, G., Zemojtel, T., Schriml, L. M., Kibbe, W. A., Schofield, P. N., Beck, T., et al. (2015). The Human Phenotype Ontology: Semantic Unification of Common and Rare Disease. <i>The American Journal of Human Genetics</i>, .</span>   <a href="http://www.cell.com/ajhg/abstract/S0002-9297(15)00234-7">Link</a>  </li>
<li><span id="pmid26092691">Haendel, M. A., Vasilevsky, N., Brush, M., Hochheiser, H. S., Jacobsen, J., Oellrich, A., Mungall, C. J., Washington, N., <b>Kohler, S.</b>, Lewis, S. E., et al. (2015). Disease insights through cross-species phenotype comparisons. <i>Mamm. Genome</i>, .</span>   <a href="http://link.springer.com/article/10.1007%2Fs00335-015-9577-8">Link</a>  </li>
<li><span id="PMID25987567"><b>Kuchenbecker, L.</b>, Nienen, M., Hecht, J., Neumann, A. U., Babel, N., Reinert, K., and <b>Robinson, P. N.</b> (2015). IMSEQ - a fast and error aware approach to immunogenetic sequence analysis. <i>Bioinformatics</i>, .</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/25987567">Link</a>  </li>
<li><span id="PMID25315429"><b>Ibn-Salem, J.</b>, <b>Kohler, S.</b>, Love, M. I., Chung, H. R., Huang, N., Hurles, M. E., Haendel, M., Washington, N. L., Smedley, D., Mungall, C. J., et al. (2014). Deletions of chromosomal regulatory boundaries are associated with congenital disease. <i>Genome Biol.</i> 15, 423.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/25315429">Link</a>  </li>
<li><span id="PMID25473437"><b>Robinson, P. N.</b> (2014). Genomic data sharing for translational research and diagnostics. <i>Genome Med</i> 6, 78.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/25473437">Link</a>  </li>
<li><span id="PMID25280750"><b>Kohler, S.</b>, Schoeneberg, U., Czeschik, J. C., Doelken, S. C., Hehir-Kwa, J. Y., <b>Ibn-Salem, J.</b>, Mungall, C. J., Smedley, D., Haendel, M. A., and <b>Robinson, P. N.</b> (2014). Clinical interpretation of CNVs with cross-species phenotype data. <i>J. Med. Genet.</i> 51, 766–772.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/25280750">Link</a>  </li>
<li><span id="PMID25186178">Zemojtel, T., <b>Kohler, S.</b>, Mackenroth, L., <b>Jager, M.</b>, Hecht, J., <b>Krawitz, P.</b>, Graul-Neumann, L., Doelken, S., Ehmke, N., Spielmann, M., et al. (2014). Effective diagnosis of genetic disease by computational phenotype analysis of the disease-associated genome. <i>Sci Transl Med</i> 6, 252ra123.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/25186178">Link</a>  </li>
<li><span id="PMID25078397">Smedley, D., <b>Kohler, S.</b>, Czeschik, J. C., Amberger, J., Bocchini, C., Hamosh, A., Veldboer, J., Zemojtel, T., and <b>Robinson, P. N.</b> (2014). Walking the interactome for candidate prioritization in exome sequencing studies of Mendelian diseases. <i>Bioinformatics</i> 30, 3215–3222.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/25078397">Link</a>  </li>
<li><span id="PMID24699242"><b>Robinson, P. N.</b>, and Webber, C. (2014). Phenotype ontologies and cross-species analysis for translational research. <i>PLoS Genet.</i> 10, e1004268.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/24699242">Link</a>  </li>
<li><span id="PMID24677618"><b>Jager, M.</b>, Wang, K., <b>Bauer, S.</b>, Smedley, D., <b>Krawitz, P.</b>, and <b>Robinson, P. N.</b> (2014). Jannovar: a java library for exome annotation. <i>Hum. Mutat.</i> 35, 548–555.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/24677618">Link</a>  </li>
<li><span id="PMID24129430">Horn, D., Wieczorek, D., Metcalfe, K., Bari, I., Paleac, L., Cuk, M., Petkovic Ramada, D., Kruger, U., Demuth, S., Heinritz, W., et al. (2014). Delineation of PIGV mutation spectrum and associated phenotypes in hyperphosphatasia with mental retardation syndrome. <i>Eur. J. Hum. Genet.</i> 22, 762–767.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/24129430">Link</a>  </li>
<li><span id="PMID24162188"><b>Robinson, P. N.</b>, <b>Kohler, S.</b>, Oellrich, A., Wang, K., Mungall, C. J., Lewis, S. E., Washington, N., <b>Bauer, S.</b>, Seelow, D., <b>Krawitz, P.</b>, et al. (2014). Improved exome prioritization of disease genes through cross-species phenotype comparison. <i>Genome Res.</i> 24, 340–348.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/24162188">Link</a>  </li>
<li><span id="PMID24217912"><b>Kohler, S.</b>, Doelken, S. C., Mungall, C. J., <b>Bauer, S.</b>, Firth, H. V., Bailleul-Forestier, I., Black, G. C., Brown, D. L., Brudno, M., Campbell, J., et al. (2014). The Human Phenotype Ontology project: linking molecular biology and disease through phenotype data. <i>Nucleic Acids Res.</i> 42, D966–974.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/24217912">Link</a>  </li>
<li><span id="PMID24020931">Dziubianau, M., Hecht, J., <b>Kuchenbecker, L.</b>, Sattler, A., Stervbo, U., Rodelsperger, C., Nickel, P., Neumann, A. U., <b>Robinson, P. N.</b>, Mundlos, S., et al. (2013). TCR repertoire analysis by next generation sequencing allows complex differential diagnosis of T cell-related pathology. <i>Am. J. Transplant.</i> 13, 2842–2854.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/24020931">Link</a>  </li>
<li><span id="PMID23104991">Doelken, S. C., <b>Kohler, S.</b>, Mungall, C. J., Gkoutos, G. V., Ruef, B. J., Smith, C., Smedley, D., <b>Bauer, S.</b>, Klopocki, E., Schofield, P. N., et al. (2013). Phenotypic overlap in the contribution of individual genes to CNV pathogenicity revealed by cross-species computational analysis of single-gene mutations in humans, mice and zebrafish. <i>Dis Model Mech</i> 6, 358–372.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/23104991">Link</a>  </li>
<li><span id="PMID21802748">Aydin, A., Desai, N., Bernhardt, A. M., Treede, H., Detter, C., Sheikhzadeh, S., Rybczynski, M., Hillebrand, M., Lorenzen, V., Mortensen, K., et al. (2013). Ascending aortic aneurysm and aortic valve dysfunction in bicuspid aortic valve disease. <i>Int. J. Cardiol.</i> 164, 301–305.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/21802748">Link</a>  </li>
<li><span id="PMID22843981"><b>Bauer, S.</b>, <b>Kohler, S.</b>, Schulz, M. H., and <b>Robinson, P. N.</b> (2012). Bayesian ontology querying for accurate and noise-tolerant semantic searches. <i>Bioinformatics</i> 28, 2502–2508.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/22843981">Link</a>  </li>
<li><span id="PMID22573485"><b>Kohler, S.</b>, Doelken, S. C., Rath, A., Ayme, S., and <b>Robinson, P. N.</b> (2012). Ontological phenotype standards for neurogenetics. <i>Hum. Mutat.</i> 33, 1333–1339.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/22573485">Link</a>  </li>
<li><span id="PMID22504886"><b>Robinson, P. N.</b> (2012). Deep phenotyping for precision medicine. <i>Hum. Mutat.</i> 33, 777–780.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/22504886">Link</a>  </li>
<li><span id="PMID22409802">Shaikh, A., and <b>Robinson, P. N.</b> (2012). An ADEPT apology. <i>Anaesthesia</i> 67, 432–433.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/22409802">Link</a>  </li>
<li><span id="PMID22331800">Chen, C. K., Mungall, C. J., Gkoutos, G. V., Doelken, S. C., <b>Kohler, S.</b>, Ruef, B. J., Smith, C., Westerfield, M., <b>Robinson, P. N.</b>, Lewis, S. E., et al. (2012). MouseFinder: Candidate disease genes from mouse phenotype data. <i>Hum. Mutat.</i> 33, 858–866.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/22331800">Link</a>  </li>
<li><span id="PMID22168972">Lucas, D. N., <b>Robinson, P. N.</b>, and Nel, M. R. (2012). Sepsis in obstetrics and the role of the anaesthetist. <i>Int J Obstet Anesth</i> 21, 56–67.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/22168972">Link</a>  </li>
<li><span id="PMID22127862"><b>Heinrich, V.</b>, Stange, J., Dickhaus, T., Imkeller, P., Kruger, U., <b>Bauer, S.</b>, Mundlos, S., <b>Robinson, P. N.</b>, Hecht, J., and <b>Krawitz, P.</b> M. (2012). The allele distribution in next-generation sequencing data sets is accurately described as the result of a stochastic branching process. <i>Nucleic Acids Res.</i> 40, 2426–2431.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/22127862">Link</a>  </li>
<li><span id="PMID21561920"><b>Bauer, S.</b>, <b>Robinson, P. N.</b>, and Gagneur, J. (2011). Model-based gene set analysis for Bioconductor. <i>Bioinformatics</i> 27, 1882–1883.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/21561920">Link</a>  </li>
<li><span id="PMID21435219"><b>Jager, M.</b>, Ott, C. E., Grunhagen, J., Hecht, J., Schell, H., Mundlos, S., Duda, G. N., <b>Robinson, P. N.</b>, and Lienau, J. (2011). Composite transcriptome assembly of RNA-seq data in a sheep model for delayed bone healing. <i>BMC Genomics</i> 12, 158.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/21435219">Link</a>  </li>
<li><span id="PMID21305018">Ott, C. E., Grunhagen, J., <b>Jager, M.</b>, Horbelt, D., Schwill, S., Kallenbach, K., Guo, G., Manke, T., Knaus, P., Mundlos, S., et al. (2011). MicroRNAs differentially expressed in postnatal aortic development downregulate elastin via 3’ UTR and coding-sequence binding sites. <i>PLoS ONE</i> 6, e16250.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/21305018">Link</a>  </li>
<li><span id="PMID21278187">Rodelsperger, C., <b>Krawitz, P.</b>, <b>Bauer, S.</b>, Hecht, J., Bigham, A. W., Bamshad, M., Condor, B. J. de, Schweiger, M. R., and <b>Robinson, P. N.</b> (2011). Identity-by-descent filtering of exome sequence data for disease-gene identification in autosomal recessive disorders. <i>Bioinformatics</i> 27, 829–836.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/21278187">Link</a>  </li>
<li><span id="PMID21109530">Rodelsperger, C., Guo, G., Kolanczyk, M., Pletschacher, A., <b>Kohler, S.</b>, <b>Bauer, S.</b>, Schulz, M. H., and <b>Robinson, P. N.</b> (2011). Integrative analysis of genomic, functional and protein interaction data predicts long-range enhancer-target gene interactions. <i>Nucleic Acids Res.</i> 39, 2492–2502.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/21109530">Link</a>  </li>
<li><span id="PMID20802478"><b>Krawitz, P.</b> M., Schweiger, M. R., Rodelsperger, C., Marcelis, C., Kolsch, U., Meisel, C., Stephani, F., Kinoshita, T., Murakami, Y., <b>Bauer, S.</b>, et al. (2010). Identity-by-descent filtering of exome sequence data identifies PIGV mutations in hyperphosphatasia mental retardation syndrome. <i>Nat. Genet.</i> 42, 827–829.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/20802478">Link</a>  </li>
<li><span id="PMID20412080"><b>Robinson, P. N.</b>, and Mundlos, S. (2010). The human phenotype ontology. <i>Clin. Genet.</i> 77, 525–534.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/20412080">Link</a>  </li>
<li><span id="PMID20172960"><b>Bauer, S.</b>, Gagneur, J., and <b>Robinson, P. N.</b> (2010). GOing Bayesian: model-based gene set analysis of genome-scale data. <i>Nucleic Acids Res.</i> 38, 3523–3532.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/20172960">Link</a>  </li>
<li><span id="PMID20144947"><b>Krawitz, P.</b>, Rodelsperger, C., <b>Jager, M.</b>, Jostins, L., <b>Bauer, S.</b>, and <b>Robinson, P. N.</b> (2010). Microindel detection in short-read sequence data. <i>Bioinformatics</i> 26, 722–729.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/20144947">Link</a>  </li>
<li><span id="PMID19800049"><b>Kohler, S.</b>, Schulz, M. H., <b>Krawitz, P.</b>, <b>Bauer, S.</b>, Dolken, S., Ott, C. E., Mundlos, C., Horn, D., Mundlos, S., and <b>Robinson, P. N.</b> (2009). Clinical diagnostics in human genetics with semantic similarity searches in ontologies. <i>Am. J. Hum. Genet.</i> 85, 457–464.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/19800049">Link</a>  </li>
<li><span id="pmid19446815">Bieler, F. H., Ott, C. E., Thompson, M. S., Seidel, R., Ahrens, S., Epari, D. R., Wilkening, U., Schaser, K. D., Mundlos, S., and Duda, G. N. (2009). Biaxial cell stimulation: A mechanical validation. <i>J Biomech</i> 42, 1692–1696.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/19446815">Link</a>  </li>
<li><span id="PMID18950739"><b>Robinson, P. N.</b>, <b>Kohler, S.</b>, <b>Bauer, S.</b>, Seelow, D., Horn, D., and Mundlos, S. (2008). The Human Phenotype Ontology: a tool for annotating and analyzing human hereditary disease. <i>Am. J. Hum. Genet.</i> 83, 610–615.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/18950739">Link</a>  </li>
<li><span id="PMID18511468"><b>Bauer, S.</b>, Grossmann, S., Vingron, M., and <b>Robinson, P. N.</b> (2008). Ontologizer 2.0–a multifunctional tool for GO term enrichment analysis and data exploration. <i>Bioinformatics</i> 24, 1650–1651.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/18511468">Link</a>  </li>
<li><span id="PMID18371930"><b>Kohler, S.</b>, <b>Bauer, S.</b>, Horn, D., and <b>Robinson, P. N.</b> (2008). Walking the interactome for prioritization of candidate disease genes. <i>Am. J. Hum. Genet.</i> 82, 949–958.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/18371930">Link</a>  </li>
<li><span id="PMID17848398">Grossmann, S., <b>Bauer, S.</b>, <b>Robinson, P. N.</b>, and Vingron, M. (2007). Improved detection of overrepresentation of Gene-Ontology annotations with parent child analysis. <i>Bioinformatics</i> 23, 3024–3031.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/17848398">Link</a>  </li>


</ol>


