<!DOCTYPE html>
<html lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      NGS &middot; CBCB
    
  </title>

  <!-- CSS -->
  <link rel="stylesheet" href="/public/css/poole.css">
  <link rel="stylesheet" href="/public/css/syntax.css">
  <link rel="stylesheet" href="/public/css/lanyon.css">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700%7CPT+Sans:400">

  <!-- Icons -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-precomposed.png">
  <link rel="shortcut icon" href="/public/favicon.ico">

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/atom.xml">
</head>


  <body class="theme-base-0d">

    <!-- Target for toggling the sidebar `.sidebar-checkbox` is for regular
     styles, `#sidebar-checkbox` for behavior. -->

<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">

<!-- Toggleable sidebar -->
<div class="sidebar" id="sidebar">
 <!-- <div class="sidebar-item">
    <p>The CBCB website</p>
  </div>
 -->
  <nav class="sidebar-nav">
  
  
  
	<a class="sidebar-nav-item" href="/">Home</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/news">News</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/compbio">Computational Biology</a>
        
                  
             
                      
                       	<a class="sidebar-nav-item-lev2" href="/compbio-onto">Ontologies</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/compbio-ngs">NGS</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/compbio-regulargenomics">Regulatory Genomics</a>
                    
            
      
	<a class="sidebar-nav-item" href="/molbio">Molecular Biology</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/software">Software</a>
        
                  
             
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-phenomizer">Phenomizer</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-phenoviz">PhenogramViz</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-phenix">PhenIX</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-exomiser">Exomiser</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-exomewalker">ExomeWalker</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-hpo">HPO</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-ontologizer">Ontologizer</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-boqa">BOQA</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-jannovar">Jannovar</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-imseq">IMSEQ</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-other">Other</a>
                    
            
      
	<a class="sidebar-nav-item" href="/teaching">Teaching</a>
        
                  
             
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">FU-Genomics</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">FU-Medical Genomics</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">FU-Genetik</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">Student projects</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">Computer diagnostics</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">Linux Tips&Tricks</a>
                    
            
      
	<a class="sidebar-nav-item" href="/team">Team</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/publications">Publications</a>
        
                  
            
      
  
    <a class="sidebar-nav-item" href="https://github.com/charite">GitHub</a>
    <span class="sidebar-nav-item">Currently v1.0.0</span>
  </nav>

  <div class="sidebar-item">
    <p>
      &copy; 2015. All rights reserved.
    </p>
  </div>
</div>




    <!-- Wrap is the content to shift when toggling the sidebar. We wrap the
         content to avoid any CSS collisions with our real content. -->
    <div class="wrap">
      <div class="masthead">
        <div class="container">
          <h3 class="masthead-title">
            <a href="/" title="Home">CBCB</a>
            <small>Computational Biology at Charité Berlin</small>
          </h3>
        </div>
      </div>

      <div class="container content">
        <div class="page">
  <h1 class="page-title">NGS</h1>
  <h3 id="next-generation-sequencing">Next Generation Sequencing</h3>

<p>Sequencing techniques of the second generation (also referred to as NGS, next generation sequencing) are currently revolutionizing genetics. By means of miniaturization and high parallelization NGS-platforms produce several orders of magnitude more sequence data than standard capillary sequencing techniques. NGS enables completely new experimental designs and poses new challenges to bioinformaticians in analyzing the data. We are currently applying NGS in three different experimental settings: Reference guided resequencing for mutation screening, whole transcriptome profiling (mRNA Seq) and transcription factor binding analysis (ChIP Seq).</p>

<h3 id="mutation-screening">Mutation Screening</h3>

<p>Our NGS sequencing facility currently generates sequence data in the order of Giga bases. We use reference guided resequencing to screen large genomic regions in the order of Mega bases or Whole Exomes for disease causing mutations in patients suffering from Mendelian diseases. With elaborate Bioinformatic filtering approaches the plethora of all called variants are narrowed down to a small set of candidate mutations <a href="#PMID25186178">(Zemojtel et al., 2014)</a>. 
<img src="/assets/images/mutscreen.jpg" alt="Mutation Screening" /></p>

<h3 id="transcription-profiling">Transcription Profiling</h3>

<p>In contrast to micro array based techniques, transcription profiling via NGS allows the detection of expressed mRNAs or miRNAs in an unbiased way as the whole transcriptome is sequenced. We currently use transcription profiling to measure the differential expression of genes that play a key role in the vertebrate development.
<img src="/assets/images/transcr_profile.jpg" alt="Mutation Screening" /></p>

<h3 id="chip-seq">ChIP Seq</h3>

<p>Chromatin Immunoprecipitation followed by massively parallel sequencing (ChIP-SEQ) is a protocol that allows to identify genomewide protein-DNA interactions. In collaboration with Jochen Hecht’s group, we are using this technique to investigate DNA binding profiles of transcription factors with prominent roles in skeletal development. During embryogenesis a plethora of transcription factors orchestrate the differential expression of the genome. Currently we are combining ChIP-SEQ and expression profiles to identify candidates for direct regulation. In a second type of assay, we compare binding profiles of wildtype transcription factors and versions with single amino acid substitutions that were detected in patients with skeletal malformations to correlate the observed phenotype with an altered DNA binding and subsequent disregulation of target genes.  Mutations in transcription factors may yield a globally changed binding pattern. ChIP Seq data will help to reveil target genes and to understand the regulatory networks underlying morphogenesis.</p>

<p><img src="/assets/images/chipseq.jpg" alt="Mutation Screening" /></p>

<h4 id="references">References</h4>
<ol class="bibliography"><li><span id="PMID25186178">Zemojtel, T., <b>Kohler, S.</b>, Mackenroth, L., <b>Jager, M.</b>, Hecht, J., <b>Krawitz, P.</b>, Graul-Neumann, L., Doelken, S., Ehmke, N., Spielmann, M., et al. (2014). Effective diagnosis of genetic disease by computational phenotype analysis of the disease-associated genome. <i>Sci Transl Med</i> 6, 252ra123.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/25186178">Link</a>  </li></ol>

</div>

      </div>
    </div>

    <label for="sidebar-checkbox" class="sidebar-toggle"></label>

    <script>
      (function(document) {
        var toggle = document.querySelector('.sidebar-toggle');
        var sidebar = document.querySelector('#sidebar');
        var checkbox = document.querySelector('#sidebar-checkbox');

        document.addEventListener('click', function(e) {
          var target = e.target;

          if(!checkbox.checked ||
             sidebar.contains(target) ||
             (target === checkbox || target === toggle)) return;

          checkbox.checked = false;
        }, false);
      })(document);
    </script>
  </body>
</html>
