<!DOCTYPE html>
<html lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      News &middot; CBCB
    
  </title>

  <!-- CSS -->
  <link rel="stylesheet" href="/public/css/poole.css">
  <link rel="stylesheet" href="/public/css/syntax.css">
  <link rel="stylesheet" href="/public/css/lanyon.css">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700%7CPT+Sans:400">

  <!-- Icons -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-precomposed.png">
  <link rel="shortcut icon" href="/public/favicon.ico">

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/atom.xml">
</head>


  <body class="theme-base-0d">

    <!-- Target for toggling the sidebar `.sidebar-checkbox` is for regular
     styles, `#sidebar-checkbox` for behavior. -->

<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">

<!-- Toggleable sidebar -->
<div class="sidebar" id="sidebar">
 <!-- <div class="sidebar-item">
    <p>The CBCB website</p>
  </div>
 -->
  <nav class="sidebar-nav">
  
  
  
	<a class="sidebar-nav-item" href="/">Home</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/news">News</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/compbio">Computational Biology</a>
        
                  
             
                      
                       	<a class="sidebar-nav-item-lev2" href="/compbio-onto">Ontologies</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/compbio-ngs">NGS</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/compbio-regulargenomics">Regulatory Genomics</a>
                    
            
      
	<a class="sidebar-nav-item" href="/molbio">Molecular Biology</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/software">Software</a>
        
                  
             
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-phenomizer">Phenomizer</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-phenoviz">PhenogramViz</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-phenix">PhenIX</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-exomiser">Exomiser</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-exomewalker">ExomeWalker</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-hpo">HPO</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-ontologizer">Ontologizer</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-boqa">BOQA</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-jannovar">Jannovar</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-imseq">IMSEQ</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-other">Other</a>
                    
            
      
	<a class="sidebar-nav-item" href="/teaching">Teaching</a>
        
                  
             
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">FU-Genomics</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">FU-Medical Genomics</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">FU-Genetik</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">Student projects</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">Computer diagnostics</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">Linux Tips&Tricks</a>
                    
            
      
	<a class="sidebar-nav-item" href="/team">Team</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/publications">Publications</a>
        
                  
            
      
  
    <a class="sidebar-nav-item" href="https://github.com/charite">GitHub</a>
    <span class="sidebar-nav-item">Currently v1.0.0</span>
  </nav>

  <div class="sidebar-item">
    <p>
      &copy; 2015. All rights reserved.
    </p>
  </div>
</div>




    <!-- Wrap is the content to shift when toggling the sidebar. We wrap the
         content to avoid any CSS collisions with our real content. -->
    <div class="wrap">
      <div class="masthead">
        <div class="container">
          <h3 class="masthead-title">
            <a href="/" title="Home">CBCB</a>
            <small>Computational Biology at Charité Berlin</small>
          </h3>
        </div>
      </div>

      <div class="container content">
        <div class="posts">
  
  <div class="post">
    <h1 class="post-title">
      <a href="http://charite.github.io//2015/06/25/hpo-common/">
        The HPO for common diseases
      </a>
    </h1>

    <span class="post-date">25 Jun 2015</span>

    <p>New paper about using the HPO for common diseases <a href="http://www.cell.com/ajhg/abstract/S0002-9297(15)00234-7">is online</a>.</p>

<p>The Human Phenotype Ontology (HPO) is widely used in the rare disease community for differential diagnostics, phenotype-driven analysis of next-generation sequence-variation data, and translational research, but a comparable resource has not been available for common disease. Here, we have developed a concept-recognition procedure that analyzes the frequencies of HPO disease annotations as identified in over five million PubMed abstracts by employing an iterative procedure to optimize precision and recall of the identified terms. We derived disease models for 3,145 common human diseases comprising a total of 132,006 HPO annotations. The HPO now comprises over 250,000 phenotypic annotations for over 10,000 rare and common diseases and can be used for examining the phenotypic overlap among common diseases that share risk alleles, as well as between Mendelian diseases and common diseases linked by genomic location. The annotations, as well as the HPO itself, are freely available.</p>

  </div>
  
  <div class="post">
    <h1 class="post-title">
      <a href="http://charite.github.io//2015/06/22/crossspecies-smedley/">
        Cross-species phenotype comparisons
      </a>
    </h1>

    <span class="post-date">22 Jun 2015</span>

    <p>New paper about cross-species phenotype comparisons <a href="http://link.springer.com/article/10.1007%2Fs00335-015-9577-8">is online</a>.</p>

<p>New sequencing technologies have ushered in a new era for diagnosis and discovery of new causative mutations for rare diseases. However, the sheer numbers of candidate variants that require interpretation in an exome or genomic analysis are still a challenging prospect. A powerful approach is the comparison of the patient’s set of phenotypes (phenotypic profile) to known phenotypic profiles caused by mutations in orthologous genes associated with these variants. The most abundant source of relevant data for this task is available through the efforts of the Mouse Genome Informatics group and the International Mouse Phenotyping Consortium. In this review, we highlight the challenges in comparing human clinical phenotypes with mouse phenotypes and some of the solutions that have been developed by members of the Monarch Initiative. These tools allow the identification of mouse models for known disease-gene associations that may otherwise have been overlooked as well as candidate genes may be prioritized for novel associations. The culmination of these efforts is the Exomiser software package that allows clinical researchers to analyse patient exomes in the context of variant frequency and predicted pathogenicity as well the phenotypic similarity of the patient to any given candidate orthologous gene.</p>

  </div>
  
  <div class="post">
    <h1 class="post-title">
      <a href="http://charite.github.io//2015/06/05/varfind/">
        Incidental findings in NGS
      </a>
    </h1>

    <span class="post-date">05 Jun 2015</span>

    <p>There are still many practical challenges to the implementation of clinical genomics, including those involving incidental findings. <a href="http://www.ncbi.nlm.nih.gov/pubmed/26036857">In this article</a>, current European views on the topic are presented.</p>

  </div>
  
  <div class="post">
    <h1 class="post-title">
      <a href="http://charite.github.io//2015/05/25/imseq/">
        IMSEQ published
      </a>
    </h1>

    <span class="post-date">25 May 2015</span>

    <p>An article by Leon Kuchenbecker has just been published in Bioinformatics. You may access the work about NGS profiling of T cell receptor repertoires <a href="http://www.ncbi.nlm.nih.gov/pubmed/25987567">here</a>.</p>

  </div>
  
  <div class="post">
    <h1 class="post-title">
      <a href="http://charite.github.io//2015/05/13/phenoviz-openhelix/">
        PhenogramViz on OpenHelix
      </a>
    </h1>

    <span class="post-date">13 May 2015</span>

    <p>PhenogramViz has been selected to be presented on the OpenHelix Blog. It is the video tip of the week. <a href="http://blog.openhelix.eu/?p=21790">See here</a>.</p>

  </div>
  
</div>

<div class="pagination">
  
    <a class="pagination-item older" href="http://charite.github.io//news/page2">Older</a>
  
  
    <span class="pagination-item newer">Newer</span>
  
</div>
 
      </div>
    </div>

    <label for="sidebar-checkbox" class="sidebar-toggle"></label>

    <script>
      (function(document) {
        var toggle = document.querySelector('.sidebar-toggle');
        var sidebar = document.querySelector('#sidebar');
        var checkbox = document.querySelector('#sidebar-checkbox');

        document.addEventListener('click', function(e) {
          var target = e.target;

          if(!checkbox.checked ||
             sidebar.contains(target) ||
             (target === checkbox || target === toggle)) return;

          checkbox.checked = false;
        }, false);
      })(document);
    </script>
  </body>
</html>
