<!DOCTYPE html>
<html lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      Ontologies &middot; CBCB
    
  </title>

  <!-- CSS -->
  <link rel="stylesheet" href="/public/css/poole.css">
  <link rel="stylesheet" href="/public/css/syntax.css">
  <link rel="stylesheet" href="/public/css/lanyon.css">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700%7CPT+Sans:400">

  <!-- Icons -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-precomposed.png">
  <link rel="shortcut icon" href="/public/favicon.ico">

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/atom.xml">
</head>


  <body class="theme-base-0d">

    <!-- Target for toggling the sidebar `.sidebar-checkbox` is for regular
     styles, `#sidebar-checkbox` for behavior. -->

<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">

<!-- Toggleable sidebar -->
<div class="sidebar" id="sidebar">
 <!-- <div class="sidebar-item">
    <p>The CBCB website</p>
  </div>
 -->
  <nav class="sidebar-nav">
  
  
  
	<a class="sidebar-nav-item" href="/">Home</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/news">News</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/compbio">Computational Biology</a>
        
                  
             
                      
                       	<a class="sidebar-nav-item-lev2" href="/compbio-onto">Ontologies</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/compbio-ngs">NGS</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/compbio-regulargenomics">Regulatory Genomics</a>
                    
            
      
	<a class="sidebar-nav-item" href="/molbio">Molecular Biology</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/software">Software</a>
        
                  
             
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-phenomizer">Phenomizer</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-phenoviz">PhenogramViz</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-phenix">PhenIX</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-exomiser">Exomiser</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-exomewalker">ExomeWalker</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-hpo">HPO</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-ontologizer">Ontologizer</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-boqa">BOQA</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-jannovar">Jannovar</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-imseq">IMSEQ</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-other">Other</a>
                    
            
      
	<a class="sidebar-nav-item" href="/teaching">Teaching</a>
        
                  
             
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">FU-Genomics</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">FU-Medical Genomics</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">FU-Genetik</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">Student projects</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">Computer diagnostics</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">Linux Tips&Tricks</a>
                    
            
      
	<a class="sidebar-nav-item" href="/team">Team</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/publications">Publications</a>
        
                  
            
      
  
    <a class="sidebar-nav-item" href="https://github.com/charite">GitHub</a>
    <span class="sidebar-nav-item">Currently v1.0.0</span>
  </nav>

  <div class="sidebar-item">
    <p>
      &copy; 2015. All rights reserved.
    </p>
  </div>
</div>




    <!-- Wrap is the content to shift when toggling the sidebar. We wrap the
         content to avoid any CSS collisions with our real content. -->
    <div class="wrap">
      <div class="masthead">
        <div class="container">
          <h3 class="masthead-title">
            <a href="/" title="Home">CBCB</a>
            <small>Computational Biology at Charité Berlin</small>
          </h3>
        </div>
      </div>

      <div class="container content">
        <div class="page">
  <h1 class="page-title">Ontologies</h1>
  <p>In the last years ontologies have become an important tool for knowledge representation in biological sciences, the most prominent being the Gene Ontology. In our group we develop new algorithms for data analysis using ontologies. We also developed a new ontology called the Human Phenotype Ontology  <a href="#PMID24217912">(Kohler et al., 2014)</a>  which is now applied in several projects in our group as well as elsewhere.</p>

<h2 id="gene-ontology">Gene Ontology</h2>

<p>The Gene Ontology (GO) provides structured, controlled vocabularies and classifications for several domains of molecular and cellular biology. GO is structured into three domains, molecular function, biological process and cellular component. The terms of the GO form a directed acyclic graph (DAG), whereby individual terms are represented as nodes connected to more specific nodes by directed edges, such that each term is a more specific child of one or more parents. For instance, mismatch repair is a child of (more specific instance of) DNA repair. The Gene Ontology Annotation (GOA) Database and several other groups provide annotations for genes or gene products of over 50 species. The true-path rule is a convention which states that whenever a gene is annotated to a term it is also implicitly associated with all the less specific parents of that term.</p>

<p><img style="float: right" src="/assets/images/ontologizer.png" /></p>

<p>We collaborated with the Computational Biology group of Martin Vingron at the Max-Planck Institute for Molecular Genetics in Berlin-Dahlem to develop a novel statistical procedure for analysis of GO term overrepresentation <a href="#PMID18511468">(Bauer et al., 2008)</a>.</p>

<h2 id="human-phenotype-ontology">Human Phenotype Ontology</h2>

<p>Recently, we have tackled the often mentioned problem of missing controlled vocabularies for mining human phenotype data by developing the Human Phenotype Ontology  <a href="#PMID24217912">(Kohler et al., 2014)</a> . We have used the HPO to annotate all disease from OMIM, Orphanet, and Decipher and annotating more databases at the moment. Please refer to the <a href="http://www.human-phenotype-ontology.org">HPO website</a> for further information. We have also used the HPO to develop a new diagnostic tool called the <a href="http://compbio.charite.de/phenomizer">Phenomizer</a> <a href="#PMID19800049">(Kohler et al., 2009)</a>.</p>

<h4 id="references">References</h4>
<ol class="bibliography"><li><span id="PMID24217912"><b>Kohler, S.</b>, Doelken, S. C., Mungall, C. J., <b>Bauer, S.</b>, Firth, H. V., Bailleul-Forestier, I., Black, G. C., Brown, D. L., Brudno, M., Campbell, J., et al. (2014). The Human Phenotype Ontology project: linking molecular biology and disease through phenotype data. <i>Nucleic Acids Res.</i> 42, D966–974.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/24217912">Link</a>  </li>
<li><span id="PMID19800049"><b>Kohler, S.</b>, Schulz, M. H., <b>Krawitz, P.</b>, <b>Bauer, S.</b>, Dolken, S., Ott, C. E., Mundlos, C., Horn, D., Mundlos, S., and <b>Robinson, P. N.</b> (2009). Clinical diagnostics in human genetics with semantic similarity searches in ontologies. <i>Am. J. Hum. Genet.</i> 85, 457–464.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/19800049">Link</a>  </li>
<li><span id="PMID18511468"><b>Bauer, S.</b>, Grossmann, S., Vingron, M., and <b>Robinson, P. N.</b> (2008). Ontologizer 2.0–a multifunctional tool for GO term enrichment analysis and data exploration. <i>Bioinformatics</i> 24, 1650–1651.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/18511468">Link</a>  </li></ol>

</div>

      </div>
    </div>

    <label for="sidebar-checkbox" class="sidebar-toggle"></label>

    <script>
      (function(document) {
        var toggle = document.querySelector('.sidebar-toggle');
        var sidebar = document.querySelector('#sidebar');
        var checkbox = document.querySelector('#sidebar-checkbox');

        document.addEventListener('click', function(e) {
          var target = e.target;

          if(!checkbox.checked ||
             sidebar.contains(target) ||
             (target === checkbox || target === toggle)) return;

          checkbox.checked = false;
        }, false);
      })(document);
    </script>
  </body>
</html>
