<!DOCTYPE html>
<html lang="en-us">

  <head>
  <link href="http://gmpg.org/xfn/11" rel="profile">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta http-equiv="content-type" content="text/html; charset=utf-8">

  <!-- Enable responsiveness on mobile devices-->
  <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">

  <title>
    
      PhenogramViz &middot; CBCB
    
  </title>

  <!-- CSS -->
  <link rel="stylesheet" href="/public/css/poole.css">
  <link rel="stylesheet" href="/public/css/syntax.css">
  <link rel="stylesheet" href="/public/css/lanyon.css">
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=PT+Serif:400,400italic,700%7CPT+Sans:400">

  <!-- Icons -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/public/apple-touch-icon-precomposed.png">
  <link rel="shortcut icon" href="/public/favicon.ico">

  <!-- RSS -->
  <link rel="alternate" type="application/rss+xml" title="RSS" href="/atom.xml">
</head>


  <body class="theme-base-0d">

    <!-- Target for toggling the sidebar `.sidebar-checkbox` is for regular
     styles, `#sidebar-checkbox` for behavior. -->

<input type="checkbox" class="sidebar-checkbox" id="sidebar-checkbox">

<!-- Toggleable sidebar -->
<div class="sidebar" id="sidebar">
 <!-- <div class="sidebar-item">
    <p>The CBCB website</p>
  </div>
 -->
  <nav class="sidebar-nav">
  
  
  
	<a class="sidebar-nav-item" href="/">Home</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/news">News</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/compbio">Computational Biology</a>
        
                  
             
                      
                       	<a class="sidebar-nav-item-lev2" href="/compbio-onto">Ontologies</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/compbio-ngs">NGS</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/compbio-regulargenomics">Regulatory Genomics</a>
                    
            
      
	<a class="sidebar-nav-item" href="/molbio">Molecular Biology</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/software">Software</a>
        
                  
             
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-phenomizer">Phenomizer</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-phenoviz">PhenogramViz</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-phenix">PhenIX</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-exomiser">Exomiser</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-exomewalker">ExomeWalker</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-hpo">HPO</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-ontologizer">Ontologizer</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-boqa">BOQA</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-jannovar">Jannovar</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-imseq">IMSEQ</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/software-other">Other</a>
                    
            
      
	<a class="sidebar-nav-item" href="/teaching">Teaching</a>
        
                  
             
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">FU-Genomics</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">FU-Medical Genomics</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">FU-Genetik</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">Student projects</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">Computer diagnostics</a>
                      
                       	<a class="sidebar-nav-item-lev2" href="/teaching-">Linux Tips&Tricks</a>
                    
            
      
	<a class="sidebar-nav-item" href="/team">Team</a>
        
                  
            
      
	<a class="sidebar-nav-item" href="/publications">Publications</a>
        
                  
            
      
  
    <a class="sidebar-nav-item" href="https://github.com/charite">GitHub</a>
    <span class="sidebar-nav-item">Currently v1.0.0</span>
  </nav>

  <div class="sidebar-item">
    <p>
      &copy; 2015. All rights reserved.
    </p>
  </div>
</div>




    <!-- Wrap is the content to shift when toggling the sidebar. We wrap the
         content to avoid any CSS collisions with our real content. -->
    <div class="wrap">
      <div class="masthead">
        <div class="container">
          <h3 class="masthead-title">
            <a href="/" title="Home">CBCB</a>
            <small>Computational Biology at Charité Berlin</small>
          </h3>
        </div>
      </div>

      <div class="container content">
        <div class="page">
  <h1 class="page-title">PhenogramViz</h1>
  <p>PhenogramViz is a tool that automatically analyses and visualizes gene-to-phenotype relations for a set of genes affected by CNV of a patient and a set of HPO-terms representing the symptoms of said patient. The tool makes full use of the cross-species phenotype ontology “uberpheno” (see <a href="http://f1000research.com/articles/2-30/v2">here</a>).</p>

<p><img src="/assets/images/phenoviz.png" alt="PhenoViz" /></p>

<h2 id="video-tutorials">Video Tutorials</h2>

<p>We have a <a href="https://www.youtube.com/channel/UC-hLx_KWD5zlbf7VnV3-fBA">Youtube-Channel</a> with several tutorial videos, e.g.</p>

<ul>
  <li>How to install PhenogamViz: <a href="https://www.youtube.com/watch?v=g4cF_Xo0eUU">Video</a></li>
  <li>How to prepare your data: <a href="https://www.youtube.com/watch?v=vQhtn1lJNkY">Video</a></li>
  <li>How to get from your data to a phenogram: <a href="https://www.youtube.com/watch?v=W9a-24G_vXI">Video</a></li>
  <li>Prioritzation of CNVs: <a href="https://www.youtube.com/watch?v=A4s4_zO7mPU">Video</a></li>
  <li>Phenogram: Internal data and external links: <a href="https://www.youtube.com/watch?v=sS_RPxh64EY">Video</a></li>
  <li>Export and saving of your results: …</li>
</ul>

<p>Please not that you have to turn on the video captions in Youtube. There is no sound in these videos!</p>

<h2 id="usage">Usage</h2>

<p>Download and install <a href="http://www.cytoscape.org/download.php">Cytoscape Version 3.1.0.</a> (PhenogramViz is not compatible to earlier versions of Cytoscape)
Download the jar-file (below) and Select App-&gt;Install from file and point the program to the downloaded jar-file.</p>

<h2 id="example-data">Example data</h2>

<p>PhenogramViz requires two datasets. At first you have to define the the symptoms of the patient (i.e. a set of HPO-term-IDs). 
Second of all you have to define the set of copy-number variant regions in the patients genome.</p>

<ol>
  <li>Symptoms
    <ul>
      <li>Enter them directly in PhenogramViz</li>
      <li>Create a file with the HPO-term IDs. Must look like this: download example patient symptoms</li>
    </ul>
  </li>
  <li>Copy-number variant regions
    <ul>
      <li>Load lists of regions (defined by Chromosome +  Start + End (+ type)</li>
      <li>Load lists of genes</li>
    </ul>
  </li>
</ol>

<h2 id="download">Download</h2>

<p>Current version: <a href="http://compbio.charite.de/tl_files/groupmembers/koehler/phenogramviz/jar/phenogramviz-0.1.3.jar">PhenogramViz (0.1.3)</a> - This jar-file can then be installed in Cytoscape.</p>

<p>Previous: <a href="http://compbio.charite.de/tl_files/groupmembers/koehler/phenogramviz/jar/phenogramviz-0.1.2.jar">PhenogramViz (0.1.2)</a>.</p>

<h2 id="citation">Citation</h2>

<p>Please cite this paper when using PhenogramViz: <a href="#PMID25280750">(Kohler et al., 2014)</a></p>

<h2 id="phenotype-data">Phenotype data</h2>

<p>The cross species phenotype resource (called uberpheno) has been developed in the context of the <a href="http://monarchinitiative.org/">Monarch Initiative</a>.</p>

<h2 id="data-from-paper">Data from paper</h2>

<p>The data used in the paper can be <a href="http://compbio.charite.de/tl_files/groupmembers/koehler/phenogramviz/data/">found here</a>.</p>

<p>This contains the uberpheno ontology (crossSpeciesPheno.obo), the gene-phenotype annotations (HSgenes_crossSpeciesPhenoAnnotation.txt), and the used CNVs (testedBenignCnvs.txt, testedPathogenicCnvs.txt). For the pathogenic CNV we also show the associated HPO-terms.</p>

<h3 id="references">References</h3>

<ol class="bibliography"><li><span id="PMID25280750"><b>Kohler, S.</b>, Schoeneberg, U., Czeschik, J. C., Doelken, S. C., Hehir-Kwa, J. Y., <b>Ibn-Salem, J.</b>, Mungall, C. J., Smedley, D., Haendel, M. A., and <b>Robinson, P. N.</b> (2014). Clinical interpretation of CNVs with cross-species phenotype data. <i>J. Med. Genet.</i> 51, 766–772.</span>   <a href="http://www.ncbi.nlm.nih.gov/pubmed/25280750">Link</a>  </li></ol>

</div>

      </div>
    </div>

    <label for="sidebar-checkbox" class="sidebar-toggle"></label>

    <script>
      (function(document) {
        var toggle = document.querySelector('.sidebar-toggle');
        var sidebar = document.querySelector('#sidebar');
        var checkbox = document.querySelector('#sidebar-checkbox');

        document.addEventListener('click', function(e) {
          var target = e.target;

          if(!checkbox.checked ||
             sidebar.contains(target) ||
             (target === checkbox || target === toggle)) return;

          checkbox.checked = false;
        }, false);
      })(document);
    </script>
  </body>
</html>
